packages/storybook/stories/SystemDownView.stories.jsx

```diff
@@ -3,7 +3,7 @@ import SystemDownView from '../../react-components/src/components/SystemDownView
 import { StoryDocs } from './wc-helpers';
 
 export default {
-  title: 'Components/System down view',
+  title: 'Deprecated/System down view',
```

Comment:
`DropdownPanel`, `SystemDownView`, and `ExpandingGroup` need the text `- React` to the end of the title like the others have but otherwise looks good!

![Screenshot 2023-03-29 at 1 09 56 PM](https://user-images.githubusercontent.com/872479/228629740-a0c92ded-8a53-4769-9db1-bd8648adb50c.png)